editor/src/components/context-menu-wrapper.tsx

```diff
@@ -191,8 +193,13 @@ export class ContextMenuWrapper<T> extends ReactComponent<
 export class InspectorContextMenuWrapper<T> extends ReactComponent<
   React.PropsWithChildren<ContextMenuWrapperProps<T>>
 > {
+  static contextType = ThemeContext
```

Comment:
I believe if you insert this line here
```typescript
declare context: React.ContextType<typeof ThemeContext >
``` 
then this.context becomes well-typed, and you can remove the `any` from line 199